cli/flox-rust-sdk/src/models/environment/managed_environment.rs

```diff
@@ -554,13 +556,28 @@ impl ManagedEnvironment {
                         })?;
                 }
                 // if it still doesn't exist after fetching, error
-                if !floxmeta
+                let in_remote = floxmeta
```

Comment:
blocking: comments need to be updated. An edge case I don't think this handles (which might be worth mentioning, but probably not fixing), is that if a managed environment is in the state fixed by this PR, but it gets moved to another directory, it will be impossible to open it.